native/cocos/editor-support/spine-creator-support/Vector2.cpp

```diff
@@ -17,15 +17,15 @@ void Vector2::setX(float x) {
     this->x = x;
 }
      
-float Vector2::getX() {
+float Vector2::getX() const {
     return x;
 }
 
 void Vector2::setY(float y) {
     this->y = y;
 }
 
-float Vector2::getY() {
+float Vector2::getY() const {
```

Comment:
Add const suffix to indicate that getter will not modify the data of this class.